tutorial/managing-software.rst

```diff
@@ -0,0 +1,1611 @@
+some of the descriptions don't actually contain the text "webserver" -- like in
+this section of the list:
+
+.. code-block:: text
+
+    inotify-tools/noble 3.22.6.0-4 amd64
+      command-line programs providing a simple interface to inotify
+
+    ipcalc/noble 0.51-1 all
+      parameter calculator for IPv4 addresses
+
+    iwatch/noble 0.2.2-10 all
+      realtime filesystem monitoring program using inotify
+
+We can use ``apt show`` to inspect the description and summary details of any
+package, so let's take a closer look at ``ipcalc`` from our list:
+
+.. code-block:: bash
+
+    apt show ipcalc
+
+The summary has been replaced with ``[...]`` for brevity, but we can see that the
+text "webserver" is in the long description of the "Description" field.
+
+.. code-block:: text
+
+    Package: ipcalc
+    Version: 0.51-1
+    [...]
+    APT-Sources: http://archive.ubuntu.com/ubuntu noble/universe amd64 Packages
+    Description: parameter calculator for IPv4 addresses
+     ipcalc takes an IPv4 address and netmask and calculates the resulting
+     broadcast, network, Cisco wildcard mask, and host range. By giving a
+     second netmask, you can design sub- and supernetworks. It is also
+     intended to be a teaching tool and presents the results as
+     easy-to-understand binary values.
+     .
+     Originally, ipcalc was intended for use from the shell prompt, but a
+     CGI wrapper is provided to enable colorful HTML display through a
+     webserver.
+     You can find it in /usr/share/doc/ipcalc/examples directory.
+
+In many places, you will see reference to ``apt-get`` and ``apt-cache`` instead
+of ``apt``. Historically, the *database* part of APT was accessed using
+``apt-cache`` (e.g. ``apt-cache show ipcalc``), and the *packages* part of APT
+used ``apt-get`` (e.g. ``apt-get install ipcalc``).
+
+APT has recently been streamlined, so although it uses ``apt-get`` and
+``apt-cache`` "behind the scenes" (and these commands do still work), we don't
+need to worry about remembering which command to use -- we can use the more
+convenient ``apt`` directly. To find out more about these packages and how to
+use them (or indeed, any package in Ubuntu!) we can refer to the manual pages.
+
+Run ``man apt``, ``man apt-get`` or ``man apt-cache`` in the terminal to access
+the manuals for these packages on the command line, or view the same content in
+the `online manual pages <https://manpages.ubuntu.com>`_.
+
+Installing deb packages
+=======================
+
+For the examples for this section, we're going to use the popular webserver
+package, `Apache2 <https://httpd.apache.org/>`_.
+
+APT gives us a lot of details about what will be included in the installation,
+and it's always important to understand the implications of a command
+*before* we run it. We'll be taking a close look at the details APT gives us,
+so we need to be careful in this section.
+
+When we run a command that asks us "``Do you want to continue? [Y/n]``", make
```

Comment:
I'm not sure. It seems like a helpful thing to mention, but since most guides and instructions a new user runs across will show them sudo commands, my intention was to talk about sudo and encourage users to slow down and read and understand what's presented (and not just spam "enter" through confirmation screens). I don't think I've ever seen the simulate flag before, so it's not commonly used in documentation. I think I'll add it, but I propose to do it at the end of the section. wdyt?